src/feature/dircache/dircache.c

```diff
@@ -1438,6 +1441,27 @@ handle_get_networkstatus_bridges(dir_connection_t *conn,
   return 0;
 }
 
+/** Helper function for GET the bandwidth file used for the next vote */
+static int
+handle_get_next_bandwidth(dir_connection_t *conn,
+                          const get_handler_args_t *args)
+{
+  (void)args;
+  log_debug(LD_DIR, "Getting next bandwidth.");
+  const or_options_t *options = get_options();
+  if (options->V3BandwidthsFile) {
+    int lifetime = 60;
+    char *bandwidth =  read_file_to_str(options->V3BandwidthsFile, 0, NULL);
+    size_t len = strlen(bandwidth);
+    write_http_response_header(conn, len, NO_METHOD, lifetime);
```

Comment:
Spec mentions `bandwidth.z`. Aren't we suppose to compress here? For instance, see how `handle_get_descriptor()` does it.

Reply:
would the connection spool be needed too?

Reply:
i guess not because it's served only 1 file